src/components/DynamicInputs.tsx

```diff
@@ -29,18 +51,23 @@ const DynamicAddressInputs: React.FC<DynamicInputsProps> = ({
       {inputArray.map((input, index) => (
         <div key={index}>
           <label>
-            Address:
+            {label}:
             <input
+              id={`${INPUT_ID[label]}-${index}`}
```

Comment:
I believe adding these will make e2e testing easier when we want to interact with this component

Reply:
totally fair!